[2024-02-25 17:21:29,229][ERROR]<airtest> Traceback (most recent call last):
  File "app\plugins\devicepool\abstract\device.py", line 84, in run
  File "app\plugins\devicepool\android\device.py", line 214, in connect
Exception: This is a 'offline' device

[2024-02-25 17:21:32,286][ERROR]<airtest> Traceback (most recent call last):
  File "app\plugins\devicepool\abstract\device.py", line 84, in run
  File "app\plugins\devicepool\android\device.py", line 214, in connect
Exception: This is a 'offline' device

[2024-02-25 17:21:37,926][ERROR]<airtest> Traceback (most recent call last):
  File "app\plugins\devicepool\abstract\device.py", line 84, in run
  File "app\plugins\devicepool\android\device.py", line 214, in connect
Exception: This is a 'offline' device

[2024-02-25 17:21:38,341][ERROR]<airtest> Traceback (most recent call last):
  File "app\plugins\devicepool\abstract\device.py", line 84, in run
  File "app\plugins\devicepool\android\device.py", line 214, in connect
Exception: This is a 'offline' device

[2024-02-25 17:21:38,742][ERROR]<airtest> Traceback (most recent call last):
  File "app\plugins\devicepool\abstract\device.py", line 84, in run
  File "app\plugins\devicepool\android\device.py", line 214, in connect
Exception: This is a 'offline' device

[2024-02-25 17:21:39,197][ERROR]<airtest> Traceback (most recent call last):
  File "app\plugins\devicepool\abstract\device.py", line 84, in run
  File "app\plugins\devicepool\android\device.py", line 214, in connect
Exception: This is a 'offline' device

[2024-02-25 17:23:43,533][ERROR]<airtest> Traceback (most recent call last):
  File "app\plugins\devicepool\abstract\device.py", line 84, in run
  File "app\plugins\devicepool\android\device.py", line 214, in connect
Exception: This is a 'offline' device

[2024-02-25 17:23:50,756][ERROR]<airtest> Traceback (most recent call last):
  File "app\plugins\devicepool\abstract\device.py", line 84, in run
  File "app\plugins\devicepool\android\device.py", line 214, in connect
Exception: This is a 'offline' device

[2024-02-25 17:23:53,525][ERROR]<airtest> Traceback (most recent call last):
  File "app\plugins\devicepool\abstract\device.py", line 84, in run
  File "app\plugins\devicepool\android\device.py", line 214, in connect
Exception: This is a 'offline' device

[2024-02-25 17:24:24,100][ERROR]<airtest> Traceback (most recent call last):
  File "app\plugins\devicepool\abstract\device.py", line 84, in run
  File "app\plugins\devicepool\android\device.py", line 214, in connect
Exception: This is a 'offline' device

[2024-02-25 17:24:27,717][ERROR]<airtest> Traceback (most recent call last):
  File "app\plugins\devicepool\abstract\device.py", line 84, in run
  File "app\plugins\devicepool\android\device.py", line 214, in connect
Exception: This is a 'offline' device

[2024-02-25 17:24:28,252][ERROR]<airtest> Traceback (most recent call last):
  File "app\plugins\devicepool\abstract\device.py", line 84, in run
  File "app\plugins\devicepool\android\device.py", line 214, in connect
Exception: This is a 'offline' device

[2024-02-25 17:24:28,478][ERROR]<airtest> Traceback (most recent call last):
  File "app\plugins\devicepool\abstract\device.py", line 84, in run
  File "app\plugins\devicepool\android\device.py", line 214, in connect
Exception: This is a 'offline' device

[2024-02-25 17:24:28,645][ERROR]<airtest> Traceback (most recent call last):
  File "app\plugins\devicepool\abstract\device.py", line 84, in run
  File "app\plugins\devicepool\android\device.py", line 214, in connect
Exception: This is a 'offline' device

[2024-02-25 17:24:34,764][ERROR]<airtest> Traceback (most recent call last):
  File "app\plugins\devicepool\abstract\device.py", line 84, in run
  File "app\plugins\devicepool\android\device.py", line 214, in connect
Exception: This is a 'offline' device

[2024-02-25 17:24:54,187][ERROR]<airtest> Traceback (most recent call last):
  File "airtest\core\android\adb.py", line 413, in shell
    out = self.raw_shell(cmd)
  File "airtest\core\android\adb.py", line 371, in raw_shell
    out = self.cmd(cmds, ensure_unicode=False)
  File "airtest\core\android\adb.py", line 236, in cmd
    raise AdbError(stdout, stderr)
airtest.core.error.AdbError: stdout[b''] stderr[b"adb.exe: device '7cbc1b6a' not found\r\n"]

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "app\plugins\devicepool\abstract\device.py", line 84, in run
  File "app\plugins\devicepool\android\device.py", line 207, in connect
  File "airtest\core\android\android.py", line 819, in display_info
    self._display_info = self.get_display_info()
  File "airtest\core\android\android.py", line 838, in get_display_info
    return self.adb.get_display_info()
  File "airtest\core\android\adb.py", line 1021, in get_display_info
    orientation = self.getDisplayOrientation()
  File "airtest\core\android\adb.py", line 1178, in getDisplayOrientation
    output = self.shell('dumpsys input')
  File "airtest\core\android\adb.py", line 415, in shell
    raise AdbShellError(err.stdout, err.stderr)
airtest.core.error.AdbShellError: stdout[b''] stderr[b"adb.exe: device '7cbc1b6a' not found\r\n"]

[2024-02-25 17:24:54,586][ERROR]<airtest.core.android.cap_methods.screen_proxy> b''
[2024-02-25 17:24:54,587][ERROR]<airtest.core.android.cap_methods.screen_proxy> b"adb.exe: device '7cbc1b6a' not found\r\n"
[2024-02-25 17:24:54,587][ERROR]<airtest.core.android.cap_methods.screen_proxy> Minicap setup up failed!
[2024-02-25 17:24:54,732][ERROR]<airtest.core.android.cap_methods.screen_proxy> b''
[2024-02-25 17:24:54,732][ERROR]<airtest.core.android.cap_methods.screen_proxy> b"adb.exe: device '7cbc1b6a' not found\r\n"
[2024-02-25 17:24:54,732][ERROR]<airtest.core.android.cap_methods.screen_proxy> Javacap setup up failed!
[2024-02-25 17:24:54,768][ERROR]<airtest.core.android.cap_methods.screen_proxy> b''
[2024-02-25 17:24:54,768][ERROR]<airtest.core.android.cap_methods.screen_proxy> b"adb.exe: device '7cbc1b6a' not found\r\n"
[2024-02-25 17:24:54,769][ERROR]<airtest.core.android.cap_methods.screen_proxy> AdbCap setup up failed!
[2024-06-07 10:08:35,475][ERROR]<airtest.core.android.cap_methods.minicap> minicap header is None
[2024-06-07 12:06:43,696][ERROR]<airtest.core.android.cap_methods.minicap> minicap header is None
[2024-06-07 12:06:46,711][ERROR]<airtest.core.android.cap_methods.minicap> minicap header is None
[2024-06-07 12:19:12,841][ERROR]<airtest.core.android.cap_methods.minicap> minicap header is None
[2024-06-07 12:19:15,852][ERROR]<airtest.core.android.cap_methods.minicap> minicap header is None
[2024-06-07 12:19:18,855][ERROR]<airtest.core.android.cap_methods.minicap> minicap header is None
[2024-06-07 12:19:21,867][ERROR]<airtest.core.android.cap_methods.minicap> minicap header is None
[2024-06-07 12:19:24,877][ERROR]<airtest.core.android.cap_methods.minicap> minicap header is None
[2024-06-07 12:46:15,828][ERROR]<airtest.core.android.cap_methods.minicap> minicap header is None
[2024-06-07 12:59:47,307][ERROR]<airtest.core.android.cap_methods.minicap> minicap header is None
[2024-06-07 13:57:01,108][ERROR]<airtest.core.android.cap_methods.minicap> minicap header is None
[2024-07-02 00:50:05,479][ERROR]<airtest.core.android.cap_methods.screen_proxy> b''
[2024-07-02 00:50:05,480][ERROR]<airtest.core.android.cap_methods.screen_proxy> b'PID: 14629\r\nINFO: Using projection 720x1600@720x1600/1\r\nINFO: (external/MY_minicap/src/minicap_29.cpp:241) Creating SurfaceComposerClient\r\nINFO: (external/MY_minicap/src/minicap_29.cpp:244) Performing SurfaceComposerClient init check\r\nINFO: (external/MY_minicap/src/minicap_29.cpp:255) Creating virtual display\r\nINFO: (external/MY_minicap/src/minicap_29.cpp:261) Creating buffer queue\r\nINFO: (external/MY_minicap/src/minicap_29.cpp:264) Setting buffer options\r\nINFO: (external/MY_minicap/src/minicap_29.cpp:268) Creating CPU consumer\r\nINFO: (external/MY_minicap/src/minicap_29.cpp:272) Creating frame waiter\r\nINFO: (external/MY_minicap/src/minicap_29.cpp:276) Publishing virtual display\r\nINFO: (jni/minicap/JpgEncoder.cpp:64) Allocating 3458052 bytes for JPG encoder\r\nERROR: (external/MY_minicap/src/minicap_29.cpp:124: errno: None) Unable to lock next buffer BAD_VALUE (-22)\r\nERROR: (jni/minicap/minicap.cpp:442: errno: None) Unable to consume pending frame\r\nINFO: (external/MY_minicap/src/minicap_29.cpp:291) Destroying virtual display\r\n'
[2024-07-02 00:50:05,480][ERROR]<airtest.core.android.cap_methods.screen_proxy> Minicap setup up failed!
